shell_automaton/tests/action_fuzz.rs

```diff
@@ -171,27 +172,31 @@ enum StatsActionTest {
         stats_current_head_actions::StatsCurrentHeadSentErrorAction,
     ),
     //TestStatsCurrentHeadRpcGetAction(stats_current_head_actions::StatsCurrentHeadRpcGetAction),
-    TestStatsCurrentHeadPruneAction(stats_current_head_actions::StatsCurrentHeadPruneAction),
-    TestStatsCurrentHeadRpcGetPeersAction(
-        stats_current_head_actions::StatsCurrentHeadRpcGetPeersAction,
-    ),
-    TestStatsCurrentHeadRpcGetApplicationAction(
-        stats_current_head_actions::StatsCurrentHeadRpcGetApplicationAction,
+    //TestStatsCurrentHeadPruneAction(stats_current_head_actions::StatsCurrentHeadPruneAction),
+    //TestStatsCurrentHeadRpcGetPeersAction(
+    //    stats_current_head_actions::StatsCurrentHeadRpcGetPeersAction,
+    //),
+    //TestStatsCurrentHeadRpcGetApplicationAction(
+    //    stats_current_head_actions::StatsCurrentHeadRpcGetApplicationAction,
+    //),
+    TestStatsCurrentHeadPrecheckInitAction(
+        stats_current_head_actions::StatsCurrentHeadPrecheckInitAction,
     ),
 }
 
 impl StatsActionTest {
     fn to_action(&self) -> Action {
         match self.clone() {
-            Self::TestStatsCurrentHeadReceivedAction(a) => a.into(),
+            //Self::TestStatsCurrentHeadReceivedAction(a) => a.into(),
             Self::TestStatsCurrentHeadPrecheckSuccessAction(a) => a.into(),
             Self::TestStatsCurrentHeadPrepareSendAction(a) => a.into(),
             Self::TestStatsCurrentHeadSentAction(a) => a.into(),
             Self::TestStatsCurrentHeadSentErrorAction(a) => a.into(),
             //Self::TestStatsCurrentHeadRpcGetAction(a) => a.into(),
-            Self::TestStatsCurrentHeadPruneAction(a) => a.into(),
-            Self::TestStatsCurrentHeadRpcGetPeersAction(a) => a.into(),
-            Self::TestStatsCurrentHeadRpcGetApplicationAction(a) => a.into(),
+            //Self::TestStatsCurrentHeadPruneAction(a) => a.into(),
```

Comment:
@dkuehr are these changes supposed to be in? there is more than just the github actions workflow in this PR

Reply:
These changes are fixes to very recent changes in the develop branch that broke the fuzzer. I first pushed the GitHub fuzzcheck action alone which run and detected these errors, so then I pushed the fixes and I'm waiting for the fuzzcheck action to finish. If the action succeeds it means it is working properly.